CONTRIBUTING.md

```diff
@@ -56,6 +56,18 @@ With these settings in place, you can prepare a test project using Docker:
 make docker_test_prepare
 ```
 
+If you'd like to run cloudiot test, you need to generate two certificates by the following actions:
+
+```sh
+for i in {1..2}; do
+  openssl genpkey -algorithm RSA -out rsa_private$i.pem -pkeyopt rsa_keygen_bits:2048
```

Comment:
Is there a reason we couldn't do these using the Terraform provider? https://www.terraform.io/docs/providers/tls/index.html